knife-vsphere.gemspec

```diff
@@ -14,7 +14,7 @@ Gem::Specification.new do |s|
   s.rubygems_version = "1.6.2"
   s.homepage = 'http://github.com/ezrapagel/knife-vsphere'
   s.license = 'Apache'
-  s.add_dependency('netaddr', ["~> 1.5.0"])
-  s.add_dependency('chef', [">= 0.10.0"])
-  s.add_dependency('rbvmomi', ["~> 1.5.1"])
+  s.add_dependency('netaddr', ['~> 1.5', '>= 1.5.0'])
+  s.add_dependency('chef', ['~> 12.0', '>= 12.0.3'])
+  s.add_dependency('rbvmomi', ['~> 1.8', '>= 1.8.0'])
```

Comment:
I'd love to get some time to get rspec/chefspec on this project so we don't have to guess on things like this.

Could you please revert the Chef version bump, please? It does appear that `:secret_file` is supported in version 10: https://github.com/chef/chef/blob/10-stable/chef/lib/chef/knife/data_bag_from_file.rb#L55